docs/DeveloperGuide.adoc

```diff
@@ -899,39 +902,209 @@ Use case ends.
 Use case resumes at step 2.
 
 [discrete]
-=== Use case: Add person
+=== Use case: Add member
+
+*System* : AddressBook
+
+*Actor* : Exco member
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to add a person to the list
-4.  AddressBook adds the person
+1.  Exco member requests to list members.
+2.  AddressBook shows a list of members.
+3.  Exco member requests to add a member to the list.
+4.  AddressBook adds the member.
 +
 Use case ends.
 
 *Extensions*
 
 [none]
-* 2a. The syntax of the add command is invalid.
+* 3a. The syntax of the add command is invalid.
+** 3a1. AddressBook shows an error message.
+** 3a1. AddressBook shows correct format for add command.
 +
-[none]
-** 2a1. AddressBook shows an error message
+Use case resumes at step 2.
+
+* 2b. The matric number of member already exist in AddressBook.
+** 2a1. AddressBook shows an error message.
 +
 Use case resumes at step 2.
 
 [discrete]
-=== Use case: Help
+=== Use case: Show Help
+
+*System* : AddressBook
+
+*Actor* : Member
 
 *MSS*
 
-1.  User requests help
-2.  AddressBook shows usage instructions
+1.  Member requests help.
+2.  AddressBook shows usage instructions.
 +
 Use case ends.
 
+[discrete]
 
-_{More to be added}_
+=== Use case: Sort members
+
+*System* : AddressBook
+
+*Actor* : Member
+
+*MSS*
+
+1.  Member requests to sort by specified field.
+2.  AddressBook sorts according to specified field.
+3.  AddressBook shows list of members in the requested sorted order.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. Invalid field specified.
+** 1a1. AddressBook shows an error message.
+** 1a2. AddressBook displays all possible valid fields.
++
+Use case resumes at step 1.
+
+[discrete]
+=== Use case: Exit
+
+*System* : AddressBook
+
+*Actor* : Member
+
+*MSS*
+
+1.  Member requests to exit AddressBook.
+2.  AddressBook asks for confirmation.
```

Comment:
The app shouldn't ask for confirmation. It's redundant. Once the user exits, the app should immediately exit